build/build.sh

```diff
@@ -0,0 +1,192 @@
+echo "$POSTGRESQL_SHA256SUM  $POSTGRESQL_FILE" |sha256sum -c
+
+echo
+echo "Downloading node..."
+echo "--------------------------------------------------------------------------"
+[[ -f "$NODE_FILE" ]] || wget -nv "$NODE_BIN_URL" --output-document="$NODE_FILE"
+echo "$NODE_SHA256SUM  $NODE_FILE" |sha256sum -c
+
+echo
+echo "Installing lisk-scripts..."
+echo "--------------------------------------------------------------------------"
+[[ -f "$LISK_SCRIPTS_FILE" ]] || wget -nv "$LISK_SCRIPTS_URL" --output-document="$LISK_SCRIPTS_FILE"
+echo "$LISK_SCRIPTS_SHA256SUM  $LISK_SCRIPTS_FILE" |sha256sum -c
+if [ ! -f "$LISK_SCRIPTS_DIR/finished" ]; then
+	rm -rf "$LISK_SCRIPTS_DIR"
+	tar xf "$LISK_SCRIPTS_FILE"
+	touch "$LISK_SCRIPTS_DIR/finished"
+fi
+
+echo
+echo "Building lisk..."
+echo "--------------------------------------------------------------------------"
+if [ ! -f "$BUILD_NAME/finished" ]; then
+	rm -rf "$BUILD_NAME"
+	tar xf "lisk-$VERSION.tgz"
+	mv package "$BUILD_NAME"
+	mkdir -p "$BUILD_NAME"/{bin,lib}
+
+	# extract postgresql
+	tar xf "$POSTGRESQL_FILE" --directory="$BUILD_NAME" \
+	    --exclude=doc --exclude=include --exclude="pgAdmin 4" --exclude=stackbuilder
+
+	# copy redis binaries
+	cp -f "$REDIS_SERVER_DIR/src/$REDIS_SERVER_OUT" "$BUILD_NAME/bin/$REDIS_SERVER_OUT"
+	cp -f "$REDIS_SERVER_DIR/src/$REDIS_SERVER_CLI" "$BUILD_NAME/bin/$REDIS_SERVER_CLI"
+	strip "$BUILD_NAME/bin/$REDIS_SERVER_OUT" "$BUILD_NAME/bin/$REDIS_SERVER_CLI"
+	# create redis workding directory
+	mkdir -p "$BUILD_NAME/redis"
+
+	# copy jq binary
+	cp -f "$JQ_FILE" "$BUILD_NAME/bin/$JQ_FILE"
+	strip "$BUILD_NAME/bin/$JQ_FILE"
+	chmod +x "$BUILD_NAME/bin/$JQ_FILE"
+
+	# copy lisk "packaged" scripts
+	cp -vrf "$LISK_SCRIPTS_DIR/packaged/"* "$BUILD_NAME"
+
+	# extract nodejs
+	tar xf "$NODE_FILE" --strip-components=1 --directory="$BUILD_NAME"
+
+	pushd "$BUILD_NAME"
+
+	echo "Setting LISK_NETWORK in env.sh..."
+	echo "--------------------------------------------------------------------------"
+	echo "export LISK_NETWORK=${LISK_NETWORK}net" >>env.sh
+
+	echo "Create default custom config.json..."
+	echo "--------------------------------------------------------------------------"
+	echo '{}' >config.json
+
+	echo "Creating etc/snapshot.json..."
+	echo "--------------------------------------------------------------------------"
+	cp config.json etc/snapshot.json
+	"./bin/$JQ_FILE" '.httpPort=9000' etc/snapshot.json |sponge etc/snapshot.json
+	"./bin/$JQ_FILE" '.wsPort=9001' etc/snapshot.json |sponge etc/snapshot.json
+	"./bin/$JQ_FILE" '.logFileName="logs/lisk_snapshot.log"' etc/snapshot.json |sponge etc/snapshot.json
+	"./bin/$JQ_FILE" '.fileLogLevel="info"' etc/snapshot.json |sponge etc/snapshot.json
+	"./bin/$JQ_FILE" '.db.database="lisk_snapshot"' etc/snapshot.json |sponge etc/snapshot.json
+	"./bin/$JQ_FILE" '.peers.list=[]' etc/snapshot.json |sponge etc/snapshot.json
+	"./bin/$JQ_FILE" '.loading.loadPerIteration=101' etc/snapshot.json |sponge etc/snapshot.json
```

Comment:
I don't think hardcode the configuration logic in build process is a good idea. We should create a seperate json file for with these values and then copy/merge it here.